setup.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+
+from setuptools import setup
+
+setup(
+    name='q',
+    url='https://github.com/harelba/q',
+    license='LICENSE',
+    version='2.0.9',
+    author='Harel Ben-Attia',
+    description="Run SQL directly on CSV or TSV files",
+    author_email='harelba@gmail.com',
+    install_requires=[
+        'six==1.11.0'
+    ],
+    packages=[
+        'bin'
+    ],
+    entry_points={
+        'console_scripts': [
+            'q = bin.q_standalone:run'
```

Comment:
I think this is wrong. You don't actually have the `bin/q_standalone` file. Maybe you meant `bin/q`?

Reply:
From PyCI: https://github.com/iliapolo/pyci/blob/release/pyci/api/package/packager.py#L529

Reply:
i've added the q_standalone file after things didn't work with `bin/q`. Thought it might be related to q not having a `.py` extension (because bin is supposed to be a package - there was always an __init__ file there).

Maybe I should just rename q to q.py and check. I'll try.

Reply:
sorry, it was 'bin.q:run_standalone`, not `bin/q:run_standalone`.

Reply:
just reminded that i already have a symlink called "qtextasdata" in this folder, exactly for treating q as a package. I'll try it as well.

Reply:
with qtextasdata it did work, but now failing on the actual upload to github. checking.

Reply:
odd, the files were uploaded to github to the proper release tag, but still it fails on a broken pipe for all of them.

Reply:
it seems that the actual q script is not included in the executable when running that way.

```
Traceback (most recent call last):
  File "/private/var/folders/my/m6ynh3bn6tq06h7xr3js0z7r0000gn/T/tmp5lwbpo3h/q-6b3d9c149ec8e0e143d950a25fe54636070c083d/bin/qtextasdata.py", line 1, in <module>
NameError: name 'q' is not defined
[19736] Failed to execute script qtextasdata
```

Perhaps the filename needs to be changed to q.py anyway

Reply:
Yeah, i think using symlinks in the entry point is problematic. And i just realized that PyCI appends a `.py` to the entrypoints. I think renaming to `q.py` will do the trick.